index.html

```diff
@@ -1,373 +1,736 @@
+          <pre>
+              <code class="language-html">
+                  @import url("https://lucidui.netlify.app/navigation/navigation.css");
+                  &lt;link rel="stylesheet"
+                  href="https://lucidui.netlify.app/navigation/navigation.css">
+              </code>
+            </pre>
+
+          <div class="examples">
+            <div class="nav">
+              <h2>Shopping App</h2>
+              <ul class="nav-links-container">
+                <li class="link">Home</li>
+                <li class="link">Wishlist</li>
+                <li class="link">Cart</li>
+              </ul>
+            </div>
           </div>
-        </div>
-      </div>
-    </section>
+          <p>
+            The class name for navigation container is nav, and for the list
+            itself is nav-links-container as shown in the code below.
+          </p>
 
-    <section id='images'>
-      <h2>Images</h2>
-      <p>Responsive image and round image.</p>
-      <div class='examples'>
-        <div class="img-container">
-          <img src="https://picsum.photos/400/300" class='img-responsive'/>
-        </div>
-        <div class="round-img-container">
-          <img src="https://picsum.photos/400/300" class="round-img" />
-        </div>
-      </div>
-    </section>
+          <script type="text/plain" class="language-markup">
+            <div class="nav">
+                <h2>...</h2>
+                <ul class="nav-links-container">
+                  <li class="link">...</li>
+                  <li class="link">...</li>
+                </ul>
+              </div>
+          </script>
+        </section>
 
-    <section id='input'>
-      <p>There are 4 different types of inputs. </p>
-      <div class='examples'>
-        <div class='input-container'>
-          <input class="input input-error" placeholder='enter email-id' type-='email'/>
-          <p>invalid input</p>
-        </div>
-        <div class='input-container'>
-          <input class="input input-warning" placeholder='password' type-='text'/>
-          <p>weak password</p>
-        </div>
-        <div class='input-container'>
-          <input class="input input-success" placeholder='enter email-id' type-='email'/>
-          <p>lets go!</p>
-        </div>
-        <input class="input" placeholder='enter email-id' type-='email'/>
-      </div>
-    </section>
+        <section id="list">
+          <h2>List</h2>
+          <pre>
+              <code class="language-html">
+                  @import url("https://lucidui.netlify.app/lists/list.css");
+                  &lt;link rel="stylesheet"
+                  href="https://lucidui.netlify.app/lists/list.css">
+              </code>
+            </pre>
+
+          <div class="examples">
+            <div class="list-wrapper">
+              <div class="list no-bullet-list">
+                <div class="list-header">
+                  <h2>list header</h2>
+                  <h3>—</h3>
```

Comment:
If you want it like : list header - , then you should reduce gap between "list header" and "-" or maybe just add hyphen in h2 only.

Reply:
Actually '-' is like collapse button, I will add an icon instead of this.